### DISCUSSION TITLE
can't retrieve image id from build stream

### DISCUSSION THREAD

**Discussion by aohanhongzhi** [containers/podman#21513] [Q&A]
I want use podman in Goland  to replace docker

https://www.jetbrains.com/help/idea/podman.html#tutorial-run-podman-in-a-virtual-machine

https://www.jetbrains.com/help/go/2023.3/connect-to-devcontainer.html

```shell
podman system service --time=0 tcp:0.0.0.0:2979
```
> I want to know how to set up the up command in config file?

![image](https://github.com/containers/podman/assets/37319319/f05f0b17-bf62-4355-9ac7-0b20aa7e9e13)

![image](https://github.com/containers/podman/assets/37319319/307252ee-fb37-4150-836c-e9c15dad4894)

![image](https://github.com/containers/podman/assets/37319319/114af3ff-dfa6-4425-a5c0-72dea71896b9)

but the error happened

```
can't retrieve image id from build stream
```

**Comment by rhatdan** [comment]
This is really a podman desktop discussion rather then a podman only one.  We recommend against using TCP connections since the podman service would open a full root exploit by doing this. We usually recommend going over ssh connections for remote connections to Podman.

**Comment by DarkGhostHunter** [comment]
It still persists using the file socket and pointing out a given step build.
